cpp/ql/src/Security/CWE/CWE-497/SystemData.qll

```diff
@@ -252,14 +252,14 @@ class LogonUser extends SystemData {
  * the `regQuery` predicate concisely.
  */
 private newtype TRegQueryParameter =
-  TSubKeyName(Expr e) or
-  TValueName(Expr e) or
-  TReturnData(Expr e)
+  TSubKeyName() or
```

Comment:
Ah. Excellent!

Reply:
Yeah, I don't know why it took me so long to realize I could do it this way!

Thanks for the help and discussion.

Reply:
No problem!